data/darktableconfig.xml.in

```diff
@@ -556,8 +556,8 @@
     <name>plugins/collect/descending</name>
     <type>bool</type>
     <default>true</default>
```

Comment:
Sorry I see that only now, but maybe the default order should be ascending no ? And so default here set to false ?